mix.exs

```diff
@@ -61,7 +62,8 @@ defmodule ExDoc.Mixfile do
       ],
       files: ["formatters", "lib", "mix.exs", "LICENSE", "CHANGELOG.md", "README.md"],
       links: %{
-        "GitHub" => "https://github.com/elixir-lang/ex_doc",
+        "GitHub" => @source_url,
+        "Changelog" => "https://hexdocs.pm/ex_doc/changelog.html",
```

Comment:
On some projects we asked not to include the changelog link as this link is not versioned. But maybe it'd only be a problem on projects like Ecto & Phoenix that have completely different changelogs per major/minor versions. I think here it would be fine but I'll let @josevalim decide.

```suggestion
```